lib/portlayer/storage/volume.go

```diff
@@ -27,9 +27,13 @@ import (
 )
 
 type Disk interface {
+	//Path to this disk on the VCH
 	MountPath() (string, error)
-	DiskPath() string
-	//FIXME: Add a capacity and populate it.
+
+	//Path to the disk on the datastore
+	DiskPath() url.URL
+	//STUFF THE STRING INTO THE PATH FIELD FOR VSPHERE IMPLEMENTATION
+	//NFS PUT THE STUFF IN THE HOST.  2 3
```

Comment:
yeah this snuck itself in. it was supposed to only be a note to myself. also the ` 2 3` are clearly me trying to switch windows in spacemacs but I was in insert mode ;-P wow...